src/CalendarDate.ts

```diff
@@ -424,4 +424,11 @@ export class CalendarDate {
       )
     );
   }
+
+  /**
+   * Returns the quarter of the year that the day is in.
+   */
+  public getQuarter(): number {
```

Comment:
```suggestion
  public get quarter(): number {
```
we can use a getter here, similiar to the week.